modules/kmm-using-intree-modules.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="kmm-using-intree-modules_{context}"]
+
+= Using in-tree modules
```

Comment:
Just curious -- for search and findability reasons, could the title more clearly indicate why you would want to use a in-tree module? For example, "Running only the device plugin by using an in-tree module". I don't know this content well, so I don't know if that would resonate with the intended audience. Just a thought.

Reply:
@ShaunaDiaz How about "Using in-tree modules with the device plugin"?